promptsource/templates/super_glue/record/templates.yaml

```diff
@@ -45,6 +80,39 @@ templates:
       original_task: true
     name: choose_between
     reference: ''
+  64013fb3-1afd-4e5a-8777-b164ca3b8e18: !Template
+    answer_choices: null
+    answer_choices_key: '{% for entity in entities[:-1] %} {{ query | replace("@placeholder",
+      entity) }} ||| {% endfor %} {{ query | replace("@placeholder", entities[-1])
+      }}'
+    id: 64013fb3-1afd-4e5a-8777-b164ca3b8e18
+    jinja: "{{ passage.split(\"@highlight\")[0] }}\n\nSummary:\n\n- {{ passage.split(\"\
+      @highlight\")[1:] | join(\"\\n- \") }} \n\n ||| {% if ( answers | length ) >\
+      \ 0 %}- {{ query | replace(\"@placeholder\", answers[0]) }} {% endif %}"
```

Comment:
move "-" in the target to the input so that the targets only contains the actual target